src/main/java/util/AllureUtil.java

```diff
@@ -0,0 +1,58 @@
+package util;
+
+import com.automation.remarks.video.recorder.VideoRecorder;
+import drivers.DriverManager;
+import io.qameta.allure.Attachment;
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.testng.ITestResult;
+import java.io.File;
+import java.io.IOException;
+import static util.EncoderUtil.convertAviToMp4;
+
+public class AllureUtil {
+
+    /***
+     Attach log file to allure report
+     */
+    @Attachment(value = "Test.log", type = "text/plain")
+    public static byte[] appendLogToAllure() {
+        try {
+            return FileUtils.readFileToByteArray(new File("target/test.log"));
+        } catch (IOException e) {
+          e.printStackTrace();
+            return null;
+        }
+    }
+
+    @Attachment(value = "Record screen MP4", type = "video/mp4")
+    public static byte[] attachVideoMP4(ITestResult result) throws IOException {
+        return FileUtils.readFileToByteArray(convertAviToMp4(result.getName() + ".mp4"));
```

Comment:
I would recommend to store video file extension in a constant as well as it's used several times

Reply:
noted

Reply:
Replace ".mp4 with a constant

Reply:
done